use_cartesian: false
over_head: false
sim_mode: false
use_depth_completion: false

gripper_offset: 0.08
intermediate_z_stop: 0.7
scan_pose: [0.45, 0.0, 0.7, 0.0, 3.14, 3.14]
bad_grasp_z: 0.12                                   # if grasp is below this point, don't execute
crop_size: [140, 325, 690, 1151]                    # 

# folder names
urdf_package_name: 'panda_simulation'               # <urdf_package_name>/urdf/objects    

# frames
base_frame: 'panda_link0'
camera_frame_sim: 'panda_camera_optical_link'
canera_frame: 'camera_depth_optical_frame'

# topics
grasp_in_image_frame: "grasp_service/predict"
grasp_in_camera_frame: "coords_in_cam"
grasp_in_world_frame: "predict"

cam_info_sim: '/panda_camera/rgb/camera_info'
depth_image_sim: '/panda_camera/depth/image_raw'
rgb_image_sim: '/panda_camera/rgb/image_raw'

cam_info: '/camera/aligned_depth_to_color/camera_info'
depth_image: '/camera/aligned_depth_to_color/image_raw'
rgb_image: '/camera/color/image_raw'
depth_complete_image: '/camera/aligned_depth_to_color/depth_completed'

cropped_depth: "cropped_depth"
cropped_rgb: "cropped_rgb"